### ISSUE TITLE
Segmentation through [S] hotkey blocked by layer control hotkey

### ISSUE THREAD

**Issue by claydugo** [computational-cell-analytics/micro-sam#811]
![s](https://github.com/user-attachments/assets/97579d0b-7c00-4d34-bd89-545eb7728a1d)

I am revisiting this tool after a long hiatus and installed fresh through conda-forge this morning.

The above imaged hotkey seems to take precedence over the `Segment Object [S]` button on the right side panel.

<details>
<summary>Version</summary>

```
$ mamba list micro
List of packages in environment: "/home/clay/miniforge3/envs/mcam_dev"

  Name       Version  Build         Channel
─────────────────────────────────────────────────
  micro_sam  1.0.1    pyhd8ed1ab_0  conda-forge

```

</details>

**Comment by anwai98** [comment]
Hi @claydugo,

I just tried this, and the keybinding `S` seems to correspond with `Segment Object [S]` for me (and in case there is no prompt when I am on `prompts` layer, the tool returns a validation message when pressing `S`). I think we fixed it recently in our latest build where we ensured compatibility with napari >0.5.

@constantinpape Do you have an idea about this?

**Comment by anwai98** [comment]
Btw, I do see one potential reason why this could be happening: we made the fixes in v1.1.0 https://github.com/computational-cell-analytics/micro-sam/releases/tag/v1.1.0 (whereas your `micro-sam` package trace says it is at version `v1.0.1`). Maybe updating it to latest `micro-sam` fixes the issues?

**Comment by claydugo** [comment]
Thank you for the extremely quick response! You are correct
